wordpress-mobile/WordPress-Android
# feat: support multiple prop in gutenberg editor

In this PR I added the possibility to set multiple media when inserting to the block.
Related gutenberg-mobile PR wordpress-mobile/gutenberg-mobile#1331
CI is failing when creating PRs from fork so there is PR that is created based on this one - https://github.com/wordpress-mobile/WordPress-Android/pull/10556
To test:
At this moment we don't support gallery block so this change can not be tested without changes in gutenberg-mobile JS bundle. In the future, we will use this functionality to multiple selections of media for the gallery.

* set multiple prop to true in media-upload bloc
* add image/video block - multiple select should be enabled for selecting media from the device or WP Media
* the array of selected media should be passed to the callback function
